.github/workflows/cont_integration.yml

```diff
@@ -50,9 +50,9 @@ jobs:
           cargo update -p tokio --precise "1.38.1"
           cargo update -p tokio-util --precise "0.7.11"
       - name: Build
-        run: cargo build ${{ matrix.features }}
+        run: cargo build --workspace --exclude 'example_*' ${{ matrix.features }}
```

Comment:
We could try just doing `--all-features` 🤔

Reply:
I think in the past we ran into errors that only happened when we tested with `--no-default-features` so I'd like to keep it.

Reply:
keep which one?

Reply:
I mean keep testing with `--no-default-features` and `--all-features`. I don't mind only building the examples with the current `stable` rust though if that's the issue.

Reply:
I think it's better to keep testing with both.